packages/nitro-rpc-client/src/transport/http.ts

```diff
@@ -17,6 +17,13 @@ export class HttpTransport {
   public static async createTransport(server: string): Promise<Transport> {
     // eslint-disable-next-line new-cap
     const ws = new w3cwebsocket(`ws://${server}/subscribe`, undefined);
+
+    // throw any websocket errors so we don't fail silently
+    ws.onerror = (e) => {
+      console.error("Error with websocket connection to server");
+      throw e;
+    };
```

Comment:
Is this related to this PR?

Reply:
Slightly, this was causing the http transport to fail silently when it couldn't make a connection to the websocket. Happy to pull this off into it's own PR.